site/dat/repo/various.json

```diff
@@ -138,6 +138,38 @@
       }
     }
   },
+  {
+    "id": "PepperBox",
+    "name": "Enhanced Kafka Support with customized schema and SASL support",
+    "description": "This plugin provides an easy way to generate load on Kafka with customized schema, high throughput and with popular security protocols",
+    "screenshotUrl": "",
+    "helpUrl": "https://github.com/GSLabDev/pepper-box",
+    "vendor": "www.gslab.com",
+    "markerClass": "com.gslab.pepper.PepperBoxLoadGenerator.java",
+    "componentClasses": [
+	  "com.gslab.pepper.input.SchemaParser.java",
+	  "com.gslab.pepper.input.SchemaProcessor.java",
+	  "com.gslab.pepper.loadgen.impl.PlaintTextLoadGenerator.java",
+	  "com.gslab.pepper.loadgen.impl.SerializedLoadGenerator.java",
+	  "com.gslab.pepper.model.FieldExpressionMapping.java",
+	  "com.gslab.pepper.sampler.PepperBoxKafkaSampler.java"
+    ],
+    "versions": {
+      "v1.0": {
+        "downloadUrl": "https://github.com/GSLabDev/pepper-box/archive/v1.0.zip",
```

Comment:
ZIP file won't be handled, `downloadUrl` has to point to `.jar` file download.

Reply:
I am done with changes but I see other plugins are providing jar downloadUrl as https://jmeter-plugins.org/files/thirdparty/. Can you share the procedure/guidelines to upload file there if any?

Reply:
/thirdparty/ is something I have built once to publish some abandoned plugins. You have to manage uploads yourself, but it's quite easy with github. You can upload your JAR into any tag, there is upload feature.
But you need to beware of one requirement: your JAR filename has to follow Maven's naming convention, where version name is part of JAR filename, plus restriction on no 'v' letter. 
For simplicity, I suggest you to work by analogy with this repo: https://github.com/Blazemeter/jmeter-bzm-plugins/releases

Reply:
Thanks for information. I'll make the suggested changes about jar filename.